proposals/nnnn-sdl_on_qnx700_agl.md

```diff
@@ -0,0 +1,254 @@
+Utils component provides to all SDL layers platform agnostic interface for communication with the operation system :
+ - file system operations
+ - threads and sync primitives
+ - timers
+ - logging
+ - system resource collecting
+ - ... 
+ 
+
+### Provide Ability for automated testing 
+
+Existing automated testing tool [sdl_atf](https://github.com/smartdevicelink/sdl_atf) and 
+[scripts](https://github.com/smartdevicelink/sdl_atf_test_scripts)
+should be used for checking SDL functionality on the new platform.
+
+#### Modification in sdl_atf
+
+sdl_atf tool should be executed on host workstation, but SDL will be executed on remote virtual workstation. 
+
+For support remote automated testing of SDL, the following proposal should be implemented: https://github.com/LuxoftAKutsan/sdl_evolution/blob/remote_atf_proxy/proposals/nnnn-remote_atf_testing.md
+
+#### Modification in test scripts
+
+Some scripts should be modified to use SDL on remote workstation.  
+All operations with SDL files ( hmi_capabilities, preloaded_pt, etc ...) should be covered with wrappers that support either local or remote execution. 
+
+
+## Testing Approach
+
+The following items should be checked for all these platforms :
+ 
+ - Compilation
+ - Unit tests
+ - Automated smoke
+ - Existing features automated test cases (in case if feature is applicable on the virtual workstation)  
+
+SDL will be tested on virtual workstation for each supported platform. 
+For communication with mobile will be used **only** TCP transport.
+For communication with HMI will be used TCP/WebSockets transport.
+
+All supported platforms (Ubuntu, AGL, QNX) will share codebase for communication with mobile and HMI.  
+
+#### Manual testing
+
+Manual testing will be performed using WebHMI https://github.com/smartdevicelink/sdl_hmi
+No changes in WebHMI expected, WebHMI will be executed on real developer workstation. 
+
+Mobile application may be connected to SDL via TCP.
+
+### Automated testing.
+
+With some changes in ATF test framework and existing scripts SDL will be tested automatically on all 3 platforms. 
+
+The same test scripts will be executed for : 
+ - Ubuntu 
+ - AGL
+ - QNX
+ 
+Test coverage will include:
+  - smoke testing of SDL (basic SDL scenarios and 3 policy flows) 
+  - all delivered features that applicable for cross platform testing.
+  
+Each further delivered feature should be tested on all supported platforms.
+
+
+## Potential downsides
+
+N/A
+
+## Impact on existing code
+
+SDL core will be modified in platform specific places.
+All components that uses **utils** will be affected and need to be retested on target platform. 
+ 
+## Alternatives considered
+
+
+#### Additional transport support on AGL, QNX virtual workstation
+
+Add USB and Bluetooth transport support for virtualized QNX and AGL platforms.
+These transport adapters will not be used for real OEMs, the only place to use them is virtual workstations.
```

Comment:
Change to:
These transport adapters will not be used for real OEMs, the only place to use them will be virtual workstations.